README.md

```diff
@@ -33,33 +33,89 @@ For [conda](https://docs.conda.io/) and [spack](https://spack.io/) installation
+```
 
-```console
-$ python train.py config_file=conf/landcoverai.yaml
+### Benchmark datasets
+
+TorchGeo includes a number of [*benchmark*](https://torchgeo.readthedocs.io/en/latest/api/datasets.html#non-geospatial-datasets) datasets, datasets that include both input images and target labels. This includes datasets for tasks like image classification, regression, semantic segmentation, object detection, instance segmentation, change detection, and more.
+
+If you've used [torchvision](https://pytorch.org/vision) before, these datasets should seem very familiar. In this example, we'll create a dataset for the Cars Overhead With Context (COWC) car detection dataset. This dataset can be automatically downloaded, checksummed, and extracted, just like with torchvision.
+
+```python
+dataset = COWCDetection(root="...", split="train", download=True, checksum=True)
 ```
 
-### Download and use the Tropical Cyclone Wind Estimation Competition dataset
+This dataset can then be passed to a PyTorch data loader.
 
-This dataset is from a competition hosted by [Driven Data](https://www.drivendata.org/) in collaboration with [Radiant Earth](https://www.radiant.earth/). See [here](https://www.drivendata.org/competitions/72/predict-wind-speeds/) for more information.
+```python
+dataloader = DataLoader(dataset, batch_size=128, shuffle=True, num_workers=4)
+```
+
+The only difference between a benchmark dataset in TorchGeo and a similar dataset in torchvision is that each dataset returns a dictionary with keys for each PyTorch Tensor.
+
+```python
+for batch in dataloader:
+    image = batch["image"]
+    label = batch["label"]
 
-Using this dataset in torchgeo is as simple as importing and instantiating the appropriate class.
+    # train a model, or make predictions using a pre-trained model
+```
+
+### Geospatial datasets
+
+Many remote sensing applications involve working with [*generic*](https://torchgeo.readthedocs.io/en/latest/api/datasets.html#geospatial-datasets) geospatial data. This data can be challenging to work with due to the sheer variety of data. Geospatial imagery is often multispectral with a different number of spectral bands and spatial resolution for every satellite. In addition, each file may be in a different coordinate reference system (CRS), requiring the data to be reprojected into a matching CRS.
+
+In this example, we show how easy it is to work with geospatial data and to sample small image patches from a combination of Landsat and Cropland Data Layer (CDL) data using TorchGeo. First, we assume that the user has Landsat 7 and 8 imagery downloaded. Since Landsat 8 has more spectral bands than Landsat 7, we'll only use the bands that both satellites have in common. We'll create a single dataset including all images from both Landsat 7 and 8 data by taking the union between these two datasets.
 
 ```python
-import torchgeo.datasets
+landsat7 = Landsat7(root="...")
+landsat8 = Landsat8(root="...", bands=landsat7.bands)
```

Comment:
Just realized that this won't work because the Landsat 8 band indices are offset by 1 from the Landsat 7 indices. Will fix this.